### PR TITLE
feat: Use monospace font for the text areas

### LINKED ISSUE #15: Use a monospace font for the textareas..
What's the latest and greatest from each OS?

### PR DESCRIPTION
**Pull request by lipis**
screenshot 2019-02-21 at 17 36 09

Fixes #15

### CODE REVIEW

File: src/components/layout/Content.tsx [hunk 1]
```diff
@@ -50,7 +50,20 @@ const formatJSON = (json: string) => {
...
+  <TextField
+    {...props}
+    fullWidth
+    multiline
+    rows={4}
+    rowsMax={Infinity}
+    variant="outlined"
+    inputProps={{
+      style: {
+        fontFamily: 'SFMono-Regular, Menlo, Monaco, Consolas, "Liberation Mono", "Courier New", monospace',
```
Comment (lipis) [inline src/components/layout/Content.tsx]: Taken from https://github.com/twbs/bootstrap/blob/master/scss/_variables.scss#L274

**Comment by lipis** [comment]
Now if we could have a direct link to test it online :P /cc @bennyn

File: src/components/layout/Content.tsx [hunk 2]
```diff
@@ -50,7 +50,20 @@ const formatJSON = (json: string) => {
...
-  <TextField {...props} fullWidth multiline rows={4} rowsMax={Infinity} variant="outlined" />
+  <TextField
+    {...props}
+    fullWidth
+    multiline
+    rows={4}
+    rowsMax={Infinity}
+    variant="outlined"
+    inputProps={{
+      style: {
```
Comment (bennyn) [inline src/components/layout/Content.tsx]: Let's move the styles to the `styles` const.

**Comment by lipis** [comment]
not deployed :)

**Comment by ffflorian** [comment]
It's deployed now

**Pull request merged**